### PR TITLE
Add "Capital loans" overview page

### PR DESCRIPTION
**Pull request by DanReyLop**
Fixes https://github.com/Automattic/woocommerce-payments/issues/3453

#### Changes proposed in this Pull Request

- Added a dedicated "Capital loans" page to the main plugin navigation. That page will be displayed in the second-to-last position (before "Settings").
- That page and navigation item will only be displayed if the user has a Capital loan or had it in the past.
- The page will include a "Test mode" notice (similar to the other pages that can show test data), plus the "Active loan summary" card, plus a list of loans, displayed in tabular form.
- The list of loans won't have sorting or pagination. I think it's not needed, since merchants will have a handful of loans at most.
- Each row will be clickable, and it will take you to the "Transactions" page, filtered by the Loan ID.

#### Testing instructions

- Run https://github.com/Automattic/woocommerce-payments-server/pull/1650 (`add/all-loans-endpoint`) on the server.
- On an account that has no tapital loans, note that you don't see any change in the plugin's navigation.
- Switch to an account that has several capital loans. The more, the better.
- Note that now you see a "Capital Loans" navigation item. Click it.
- You should see a page with the Active Loan Summary card and a list of loans. Check the appearance of the page with the Figma design file.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

**Review by luizreis — CHANGES_REQUESTED** [review]
@DanReyLop - This is looking good! The loan page is 🔥 and the links from the inbox note and transaction summary work well. I've left a few comments about a regression and a couple of small UI issues that we should fix before shipping this PR.

File: client/capital/index.tsx
Comment (DanReyLop) [reply client/capital/index.tsx:117]: I think the designs were done before we decided on a consistent approach for displaying money amounts. I looked at other similar tables (transactions, deposits) and they all use the explicit format. So that's what I went with.

File: client/capital/index.tsx
Comment (DanReyLop) [reply client/capital/index.tsx:47]: Huh... can you tell me your WooCommerce and WP versions? And WooCommerce Admin, if you have it enabled? They're left-aligned in my environment, but I'll add a CSS style if your environment is "supported" and it isn't working for you.

Comment (luizreis) [reply client/capital/index.tsx:47]: Sure! I'm using WP 5.9, with WC 6.1.1 and no WC Admin plugin.

File: client/capital/index.tsx
Comment (luizreis) [reply client/capital/index.tsx:117]: That makes sense, let's keep the explicit format then 🙂

File: includes/class-wc-payments-account.php
Comment (DanReyLop) [reply includes/class-wc-payments-account.php:187]: Good catch! That was a terrible oversight.

> we should modify the overview component to use the has_active_loan from the capital data

Done in https://github.com/Automattic/woocommerce-payments/pull/3831/commits/dab3e8e50bdd53aa27c0e4c445f7661006039ef7

File: client/capital/index.tsx
Comment (DanReyLop) [reply client/capital/index.tsx:47]: As we discussed on Slack, this is a bug on newer versions of WooCommerce Admin. Adding `isNumeric` to the columns fixes the alignment, and it doesn't have any other unintended consequence (the `max-width` we saw was just for placeholders). I've added `isNumeric` on https://github.com/Automattic/woocommerce-payments/pull/3831/commits/bb2bec8a098671e730035a0f7ef238c7a01ed6a3

File: client/capital/index.tsx
Comment (DanReyLop) [reply client/capital/index.tsx:137]: Argh, all my loans had paydowns already. On , I've made it so it renders a "-" instead of "Invalid date". Whatever we decide the long-term fix for this is, it will be a server-side-only change.

@LevinMedia Any thoughts on this? "First paydown [date]" has a different meaning in the summary (the date when Stripe will start taking a cut from transactions) and in the loans list (the date of the first paydown). Should we change the label to be clear that those 2 are different dates, or change one of them to fit the other?

**Review by luizreis — APPROVED** [review]
Thanks for fixing those issues @DanReyLop. I've kept the discussion around "First paydown" unresolved for visibility, but it shouldn't block this PR.

I'm approving this, but there are quite a few tests failing on the GH actions. I wasn't able to reproduce it locally, though. We should probably check if those are related to this PR before merging. :shipit:

**Pull request merged**
